emperor/flat_plots.py

```diff
@@ -0,0 +1,337 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+
+
+def plot_3x3(ordination, offset, mapping, field, colors, null_kw=None,
+             focus_kw=None):
+    """Construct a 3 x 3 plot of PC combinations
+
+    Where X is a PCn / PCm plot, _ is a blank spot, and L is the legend.
+
+    Offset defines the starting offset for the rows. If zero, the layout
+    is:
+
+    PC1v2 PC1v3 PC1v4 BLANK
+    PC2v3 PC2v4 PC2v5 Legend
+    PC3v4 PC3v5 PC3v6 BLANK
+
+    Parameters
+    ----------
+    ordination : OrdinationResults
+        An OrdinationResults object
+    offset : int
+        A starting offset
```

Comment:
It is not exactly clear what this offset is referring to.  Is it the spacing in between subplots?